### DISCUSSION TITLE
Capital letters

### DISCUSSION THREAD

**Discussion by PhilStrass** [YOURLS/YOURLS#2977] [Q&A] [answered]
Hello,
Is it possible to configure YOURLS to accept capital letters in the output short url ?
Thanks for your advice

**Comment by dgw** [comment, marked as answer]
Yes: [set `YOURLS_URL_CONVERT` to `62`](https://yourls.org/#Config;urlsettings) in your config. You probably have this if you copied YOURLS' sample config file:

https://github.com/YOURLS/YOURLS/blob/fd938fd19742e141f95f14ef654d1501bdb63679/user/config-sample.php#L71-L75

Change that `36` to `62`, and check to make sure your existing short links (hopefully not too many yet) still behave as expected. It's not recommended to change this setting on an install that has many links (see the [FAQ entry about shortlink encoding](https://yourls.org/#FAQ;base3662)).

**Reply by mike14-2** [reply]
Hello,
Thank you very much.
I have tried this and it works with http links.
But now I still have app links that I want to use.
For example, the link is structured like this:

x-devonthink-item://930FBCF1-CB42-45B8-9092-904030947FBF

I have already set the config to accept "x-devonthink-item://".
But my problem is that YOURLs then writes all the letters after the "//" in lower case.

Can anyone else help me with this?
Thank you very much

**Reply by dgw** [reply]
I get the impression that you have ignored both the request below to add details to #3741, and [the detailed answer](https://github.com/orgs/YOURLS/discussions/3741#discussioncomment-9123234) I posted to that discussion based on what you provided here, in [your earlier comment](https://github.com/orgs/YOURLS/discussions/2977#discussioncomment-9122385) which has apparently been edited and now appears empty.

This discussion will be locked. Please continue to provide details at #3741 if the existing analysis is insufficient.

**Comment by mike14-2** [comment]


**Reply by dgw** [reply]
This is an unrelated question that should have its own thread.

**Reply by dgw** [reply]
I see you have posted before. Please add the details to #3741 demonstrating that it is not a duplicate of this thread.
